generators/client-2/templates/_webpack-dev

```diff
@@ -0,0 +1 @@
+npm run webpack-dev
```

Comment:
may be i'm naive, but then why not just call the command from npm why does it need a file?